docs/building-functions/on_cluster_build.md

```diff
@@ -49,12 +49,16 @@ git remote add origin git@github.com:my-repo/my-function.git
 ```
 4. Update the Function configuration in `func.yaml` to enable on cluster builds for the Git repository:
 ```yaml
-build: git                                          # required, specify `git` build type
-git:
-  url: https://github.com/my-repo/my-function.git   # required, git repository with the function source code
-  revision: main                                    # optional, git revision to be used (branch, tag, commit)
-  # contextDir: myfunction                          # optional, needed only if the function is not located
-                                                    # in the repository root folder
+build:
+  git:
+    url: https://github.com/my-repo/my-function.git   # required, git repository with the function source code
+    revision: main                                    # optional, git revision to be used (branch, tag, commit)
+    # contextDir: myfunction                          # optional, needed only if the function is not located in the repository root folder
+  # builderImages:                                    # optional, needed only if the runtime is golang 
+  #   pack: ghcr.io/boson-project/go-function-builder:tip
+  buildpacks: []
+  builder: ""
```

Comment:
Builder should be `"pack"`.

Reply:
but empty string probably would work too

Reply:
If these two items are intended as hints, commenting them would probably be best.

The idea here is that the code has static defaults.  Anything placed in the config file is specifically requesting that exact value, so while the first value, `[]buildpacks` is innocuous, the second, `builder=""` is, if literally translated, requesting the system to build without a builder.  In this case we do set the default if the value comes in empty like this, because building without a builder is nonsensical, but I presume this exists to give a hint to the user how to choose a builder and buildpacks, so a commented entry is the right tool for the job.